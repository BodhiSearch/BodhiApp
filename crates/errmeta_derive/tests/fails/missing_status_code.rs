#[derive(errmeta_derive::ErrorMeta)]
enum TestError {
  #[error_meta(code = "bad_request", error_type = "invalid_request")]
  BadRequest,
}

fn main() {}
