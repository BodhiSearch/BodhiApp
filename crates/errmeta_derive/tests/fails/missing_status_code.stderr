error: status not specified for variant 'BadRequest'
 --> tests/fails/missing_status_code.rs:1:10
  |
1 | #[derive(errmeta_derive::ErrorMeta)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `errmeta_derive::ErrorMeta` (in Nightly builds, run with -Z macro-backtrace for more info)
